site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Optimizing LCP using Signed Exchanges
+description: >
+  Signed exchanges are a means for web developers to drastically improve page loading speed coming from SXG referrers such as Google Search. They can provide benefit simply from being enabled by a checkbox, but you can take additional steps to get the most out of them.
+subhead: >
+  How to measure and optimize signed exchanges to get the most improvement out of them.
+date: 2022-04-15
+
+authors:
+  - twifkak
+
+tags:
+  - performance
+
+# Optional
+# Hero images should be at least 1920 x 960
+# https://developer.chrome.com/docs/handbook/how-to/add-media/
+# hero: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
+
+# Optional
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# Thumbnail images should be at least 828 x 416
+# thumbnail: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
+
+# Required if there is a hero or social image
+# alt: An alternative text description of your hero or social image.
+---
+
+<!--
+- Only use h2-h4s in markdown.
+- Make sure all of your images have `alt` text.
+- Code blocks should use triple backticks and include a language name.
+-->
+
+Signed exchanges (SXGs) are a means for web developers to improve their page speed—mainly [Largest Contentful Paint (LCP)](https://web.dev/lcp/)—by enabling other sites (currently Google Search) to [prefetch their pages](https://developers.google.com/search/docs/advanced/experience/signed-exchange) before the user clicks on the link.
+
+{% Aside %}
+SXGs are potentially useful for other use cases not covered in this article. For instance, they could enable connection resilience: a chat app could pre-download an SXG of a link sent to the user, so they could later view it while offline. See this larger list of [potential use cases](https://wicg.github.io/webpackage/draft-yasskin-wpack-use-cases.html).
+{% endAside %}
+
+It is possible to make web pages that completely eliminate the network from the critical path to rendering the page, when prefetched! On 4G, this page load [goes from 6s to 0.9s](https://www.webpagetest.org/video/compare.php?tests=220329_BiDc1V_FJG,220329_AiDcB7_HB0&medianMetric=LCP). What remains is bound mostly by CPU:
+
+{% Video src="video/rULxC7pPw3PFS4o9xr7v8isFmCv1/5IRBMSsCZwihnVIVK4gS.mp4",
+         controls=true,
+         poster="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/4ttTTum2aZuNZL04vYmI.jpg" %}
```

Comment:
```suggestion
<figure>
  {% Video src="video/rULxC7pPw3PFS4o9xr7v8isFmCv1/5IRBMSsCZwihnVIVK4gS.mp4", controls=true, poster="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/4ttTTum2aZuNZL04vYmI.jpg" %}
</figure>
```

Reply:
Since these are figures, they should be encased in a `<figure>` element for those on assistive technology.